<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnBar</class>
 <widget class="QWidget" name="ColumnBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="BarBeginRowEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起始行(默认为2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>截止行(默认底行)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="BarEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ColumnBar2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BarEndRowEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
